.github/internal/install-shinyverse/action.yaml

```
@@ -56,38 +74,53 @@ runs:
       run: |
         # Get R version like `"4.2"`
         short_r_version <- sub("\\.\\d$", "", "${{ inputs.r-version }}")
-        # For R versions >= 4.2, use Rtools v42
+        r_version <- package_version("${{ inputs.r-version }}")
+        # For R versions >= 4.4.*, use Rtools v44
+        # For R versions == 4.3.*, use Rtools v43
+        # For R versions == 4.2.*, use Rtools v42
         # Otherwise leave blank
-        if (
-          package_version("${{ inputs.r-version }}") >= package_version("4.2")
-        ) {
-          message("Using Rtools v42")
-          cat("value=42\n", file = Sys.getenv("GITHUB_OUTPUT"), sep = "", append = TRUE)
+
+        # R version >= 4.2
+        value <-
+          if (short_r_version == "4.4") {
+            "44"
+          } else if (short_r_version == "4.3") {
+            "43"
+          } else if (short_r_version == "4.2") {
+            "42"
+          } else {
+            ""
+          }
```

Would it make sense to try and future-proof this?

```suggestion
        value <- if (r_version >= "4.2") {
          sub(".", "", short_r_version, fixed = TRUE)
        } else {
          ""
        }
```

---

I believe we can drop it entirely. 

https://github.com/rstudio/shiny-workflows/commit/4a08b567b41417f00a19b0f63897ba89e91282ba 

^^ will be doing automatic support for the rtools version

---

Dropped. **resolving